index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- font CDN -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- HEADER SECTION -->
+    <div class="header">
+        <div class="header__logo">David</div>
+        <div class="header__menu-btn">
+            <div class="menu-btn__ham"></div>
+        </div>
```

Comment:
- [ ]  Please consider wrapping the menu button with a `nav` tag instead of a `div` tag as the menu button which is known as the hamburger menu consists of navigation links and also to improve the semantic structure of your HTML😄. You can read more on [SEMANTIC HTML](https://www.w3schools.com/html/html5_semantic_elements.asp). See my suggestions below.

```suggestion
        <nav class="header__menu-btn">
            <div class="menu-btn__ham"></div>
        </nav>
```